docs/eventing/getting-started.md

```diff
@@ -19,299 +21,377 @@ Use this guide to learn how to create, send, and verify events in Knative. The s
 
 To complete this guide, you will need the following installed and running:
 
-- A [Kubernetes cluster](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/) running v1.15 or higher.
+- A
```

Comment:
I'd remove this whole installation bit here and just point to any-kubernetes-cluster.md. They have drifted in the past.